File: Data Structures & Algorithms/two-integer-sum/submission-2.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        for i in range(len(nums)):
            difference = target - nums[i]
            if(difference in nums[i+1:]):
                return [i,nums.index(difference, i+1)]

            

File: Data Structures & Algorithms/binary-search/submission-2.cpp
class Solution {
public:
    int search(vector<int>& nums, int target) {
        
        if(nums.size() == 0) return -1;
        
        for(int i = 0; i <= nums.size() - 1; i++){
            if(nums[i] == target)
                return i;
        }
        return -1;
    }
};
